service/schema.py

```diff
@@ -0,0 +1,25 @@
+import json
+from jsonschema import validate
+
+schema = {
+    "type": "object",
+    "properties": {
+        "item_one": {"type": "string"},
+        "item_two": {"type": "number"}
+    }
+}
+
+
+def validate_schema(message_payload):
+    """
+    Validate message queue payload against json schema
+        - message_payload: queue item message
+    """
+    message_payload = json.dumps(message_payload)
+    try:
+        validate(instance=message_payload, schema=schema)
+        return True
+    except Exception as err:
+        # make sure we log errors for debugging purposes
```

Comment:
needs to decide whether to mark the message as received or not based on config, you'll probably need to update code a bit to make that work.

Reply:
@mikeAdamss - so ill put a 'mark_message_recieved' in config for this approach if youre happy with this and pass it through